python/vineyard/contrib/dask/dask.py

```diff
@@ -25,21 +25,58 @@
 
 import dask.array as da
 import dask.dataframe as dd
+import numpy as np
+import pandas as pd
 
 
 def dask_array_builder(client, value, builder, **kw):
-    # TODO: build dask.array to vineyard::GlobalTensor
-    pass
+    def put_partition(v, block_id=None):
+        client = vineyard.connect()
+        obj_id = client.put(v, partition_index=block_id)
+        client.persist(obj_id)
+        return np.array([[int(obj_id)]])
+
+    _ = Client(kw['dask_scheduler'])  #enforce distributed scheduling
+    ids = value.map_blocks(put_partition, dtype=int).compute().flatten()
+    meta = ObjectMeta()
+    meta['typename'] = 'vineyard::GlobalTensor'
+    meta.set_global(True)
+    meta['partitions_-size'] = len(ids)
+
+    for i, obj_id in enumerate(ids):
+        meta.add_member('partitions_-%d' % i, vineyard.ObjectID(obj_id))
+
+    gtensor_meta = client.create_metadata(meta)
+    client.persist(gtensor_meta)
+    return gtensor_meta
 
 
 def dask_dataframe_builder(client, value, builder, **kw):
-    # TODO: build dask.dataframe to vineyard::GlobalDataFrame
-    pass
+    def put_partition(v, partition_info=None):
+        client = vineyard.connect()
+        obj_id = client.put(v, partition_index=(partition_info['number'], 0))
+        client.persist(obj_id)
+        return pd.DataFrame([{'no': partition_info['number'], 'id': int(obj_id)}])
```

Comment:
Could we return a `vineyard.ObjectID` directly? Why we need to wrap it into a dataframe?